src/controllers/OrganisationsController.js

```diff
@@ -122,6 +53,13 @@ const fetchDatasetInfo = fetchOne.bind({
   result: 'dataset'
 })
 
+// const fetchOrgInfoWithStatGeo = fetchOne.bind({
+//   query: ({ params }) => {
+//     return /* sql */ `SELECT name, organisation, statistical_geography FROM organisation WHERE organisation = '${params.lpa}'`
+//   },
+//   result: 'orgInfo'
+// })
+
```

Comment:
Should we just remove this? if it's not being used, then there's no point of keeping unused code around

Reply:
They're used in the next PR.